descriptors/ModuleDescriptor-template.json

```diff
@@ -135,6 +135,24 @@
             "consortia.user-tenants.item.get"
           ],
           "modulePermissions": []
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/consortia-configuration",
```

Comment:
I am thinking consoritumId should be included in here. so that there is potential to get config for specific consortium. Do you see where I am getting at?

Reply:
will it be possible to have several consortia for one tenant?

Reply:
/consortia-configuration will be called by all tenants, it means they won't have consortium entity in their schemas 

and additionally from other modules that need to know about central tenant, if we introduced consoritumId in parameters, we would need to make 2 requests to get info about central tenant:
1. GET /consortia - it means POST /consortia should be called for each tenant during initial set up
2. GET /consortia-configuration

Reply:
> will it be possible to have several consortia for one tenant?

I would like leave the avenue of having several consortia in one cluster. API contracts are going to be harder to change once set so the intent was to not close that gap.

Prospective consortia are going to be large enough to have their own clusters. So multiple consortia in a single cluster might not happen in practice. We can move forward with the requirements we have for now.

Reply:
I believe what @azizbekxm did doesn't contradict with possibility to have several consortia in one cluster and even help to achieve it if one tenant always belongs to one consortium

e.g. we have 2 consortia in one cluster
consortium A:
centralTenantA (has consortium object, consortia-configuration - centralTenantA)
tenantA (no consortium object, consortia-configuration - centralTenantA)
tenantB (no consortium object, consortia-configuration - centralTenantA)

consortium B:
centralTenantB (has consortium object, consortia-configuration - centralTenantB)
tenantA (no consortium object, consortia-configuration - centralTenantB)
tenantB (no consortium object, consortia-configuration - centralTenantB)

Reply:
but it should be tested, it's theory for now

Reply:
Thanks for jogging my thought process, it should still be possible to have multiple consortia in the cluster.